hosting/models.py

```diff
@@ -194,6 +197,13 @@ def get_absolute_url(self):
     def __str__(self):
         return ", ".join([self.city, force_text(self.country.name)]) if self.city else force_text(self.country.name)
 
+    def display_family_members(self):
+        family_members = self.family_members.exclude(pk=self.owner.pk).order_by('birth_date')
+        return ", ".join(fm.repr() for fm in family_members)
+
+    def display_conditions(self):
+        return ", ".join(c.__str__() for c in self.conditions.all())
```

Comment:
kial ne `abbr` kiu laŭdifine estas _Official abbreviation as used in the book_, dum `name` estas la hom-legebla teksto povanta ŝanĝiĝi ?